packages/type/src/inquirer.mts

```diff
@@ -32,4 +32,4 @@ export type Context = {
 export type Prompt<Value, Config> = (
   config: Config,
   context?: Context,
-) => CancelablePromise<Value>;
+) => Promise<Value> & { cancel: () => void };
```

Comment:
I'll completely remove CancelablePromise in this case; getting it closer to Node 20 deprecation.